task_scheduler.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime as dt
+
+class Task:
+    def __init__(self, name: str, duration: int, deadline: str, completed = False) -> None:
+        # tasks begin as soon as they are added
+        self.name = name
+        self.duration = duration
+        # convert deadline from 'YYYY-MM-DD HH:MM' to datetime object
+        self.deadline = dt.strptime(deadline, '%Y-%m-%d %H:%M')
+        self.completed = completed
+
+class TaskScheduler:
+
+    STATUS_MESSAGE = {
+        
+    }
+
+    def __init__(self, tasks = None) -> None:
+        if tasks is None:
+            tasks = {}
+        self.tasks = tasks
```

Comment:
```suggestion
    def __init__(self, tasks = {}) -> None:
        self.tasks = tasks
```
The optional parameter does what you did manually, just cleaner and easier.